routing/validation_barrier.go

```diff
@@ -3,12 +3,17 @@ package routing
 import (
 	"fmt"
 	"sync"
+	"time"
 
 	"github.com/lightningnetwork/lnd/channeldb"
 	"github.com/lightningnetwork/lnd/lnwire"
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+// waitingParentSignalTimeout defines the time to wait before the dependant
+// gives up waiting for its parent's signal to either allow or deny.
+const waitingParentSignalTimeout = 60 * time.Second
```

Comment:
I'm not too familiar with the gossip handling so forgive my ignorance: But how did you determine the value for this? Just something that makes sense? Is there any benefit in making this configurable?

Reply:
Very good question! Replied below to bring more discussion.